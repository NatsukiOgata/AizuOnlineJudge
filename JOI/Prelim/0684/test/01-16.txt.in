80 60
71 95 4 11 8 61 76 29 81 43 44 36 73 54 57 100 34 19 63 74 53 43 78 6 20 89 40 16 10 7 56 40 28 15 46 19 100 53 86 29 55 69 64 4 52 67 35 35 37 94 47 10 63 43 10 32 9 70 4 65 62 53 86 83 90 10 76 26 38 60 42 94 58 18 81 72 15 96 60 27
98 80 11 28 64 77 4 42 35 93 90 32 45 32 70 97 97 62 95 15 12 7 51 1 72 49 91 47 6 22 23 85 85 83 30 8 96 23 83 53 27 19 21 76 81 30 68 31 67 37 55 66 3 41 66 75 30 40 8 37
