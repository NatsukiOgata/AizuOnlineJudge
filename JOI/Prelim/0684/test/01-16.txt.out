4
6
7
8
11
15
19
27
28
32
35
37
40
42
47
53
55
62
64
67
70
72
76
81
83
90
95
96
